api/components/Examples/Agriculture_services/on_search/on_search.json

```diff
@@ -0,0 +1,457 @@
+        "short_desc": "Agri Service Provider",
+        "long_desc": "Agri Service Provider",
+        "images": [
+          {
+            "url": "https://abc.com/images/1-shop-img"
+          }
+        ]
+      },
+      "providers": [
+        {
+          "id": "P1",
+          "descriptor": {
+            "name": "ABC service provider store",
+            "code": "P001",
+            "short_desc": "ABC service provider store",
+            "long_desc": "ABC service provider store",
+            "images": [
+              {
+                "url": "https://abc.com/images/1-shop-img"
+              }
+            ]
+          },
+          "rating": "4.4",
+          "ttl": "PT1D",
+          "locations": [
+            {
+              "id": "L1",
+              "gps": "12.967555,77.749666",
+              "address": "Jayanagar 4th Block",
+              "city": {
+                "code": "std:080",
+                "name": "Bengaluru"
+              },
+              "state": {
+                "code": "KA"
+              },
+              "country": {
+                "code": "IND"
+              },
+              "area_code": "560076"
+            }
+          ],
+          "creds": [
+            {
+              "id": "ESG-12345678",
+              "type": "Certificate",
+              "desc": "Samsung Authorized Service provider",
+              "url": "https://abcd.cdn.com/images/bage-img"
+            }
+          ],
+          "tags": [
+            {
+              "descriptor": {
+                "code": "serviceability"
+              },
+              "list": [
+                {
+                  "descriptor": {
+                    "code": "location"
+                  },
+                  "value": "L1"
+                },
+                {
+                  "descriptor": {
+                    "code": "category"
+                  },
+                  "value": "AGR12-*"
+                },
+                {
+                  "descriptor": {
+                    "code": "type"
+                  },
+                  "value": "13"
+                },
+                {
+                  "descriptor": {
+                    "code": "val"
+                  },
+                  "value": "{\"type\": \"FeatureCollection\", \"features\": [{\"type\": \"Feature\", \"properties\": {}, \"geometry\": {\"coordinates\": [[[77.175571, 28.675927], [77.128737, 28.600972], [77.446934, 28.545325], [77.175571, 28.675927]]], \"type\": \"Polygon\"}}]}"
+                },
+                {
+                  "descriptor": {
+                    "code": "unit"
+                  },
+                  "value": "geojson"
+                }
+              ]
+            }
+          ],
+          "items": [
+            {
+              "id": "I1",
+              "parent_item_id": "PI1",
+              "descriptor": {
+                "name": "Soil Testing - Primary & Secondary",
+                "code": "UPC code",
```

Comment:
Is it a UPC code for services items, or do we have any unique ids to agri services.